tutorials/en/vectara-beginner-app-tutorial-showcase-the-creation-of-vectara-app-in-legal-use-case.mdx

```diff
@@ -0,0 +1,390 @@
+Author Username: "sanchayt743"
+---
+
+## 1. Introduction to the Vectara Ecosystem: 🌌
+
+Welcome to the realm of Vectara, a platform where search transcends to new horizons powered by Generative AI. Here, I will unravel the essence of the Vectara ecosystem, its fundamental workings, and provide a visual journey through Vectara's official materials to enhance your comprehension.
+
+### 🚀 Overview of Vectara Ecosystem
+
+Vectara is on a mission to redefine how users interact with data and knowledge, facilitating a seamless journey from a user query to the most relevant response. The platform harbors a complete, but easy to customize, search and summarization pipeline, making it an API-based easy-to-use platform to build applications powered by semantic search and Generative AI. Through Vectara, developers are empowered to create GenAI applications with a state-of-the-art retrieval engine and summarization capability, thus elevating the user experience to a realm where questions meet precise answers.
+
+### 🛠 Fundamental Workings and Workflow
+
+The heartbeat of Vectara is its pure neural search platform enriched with production-ready natural language processing. The workflow is simple, yet powerful:
+
+1. **Data Ingestion**: Ingest your data into Vectara's corpus using the Indexing API.
+2. **Query Execution**: Utilize the Search API to run queries against the indexed data, retrieving highly relevant information swiftly.
+
+The beauty of Vectara lies in its API-addressable platform, which is a canvas for developers to paint their GenAI solutions, embedding them within their applications.
+
+---
+### 🎨 Dive into Vectara's Console
+
+To truly grasp the potential of Vectara, let's delve into its console which is the epicenter of managing your account:
+
+- **Creating Corpora**: Jumpstart your journey by establishing a corpus, your data's secure haven, ready for querying. Here’s how to navigate this straightforward process:
+  - **Name Your Corpus**: Give your corpus a unique identifier.
+  - **Provide a Description**: Briefly describe the purpose or content of your corpus.
+  - **Choose an Embedding Model**: Select the embedding model that best aligns with your needs.
+  - **Specify Filter Attributes (Optional)**: You have the option to add filter attributes for additional refinement.
+  
+  Just like that, your corpus is configured and ready to receive data!
+
+- **API Access Management**: Vectara empowers you with the tools to seamlessly manage API access. Utilize the `API access` tab, visible in the sidebar once the requisite permissions are granted, to effortlessly create and manage your API keys and app clients. This is your gateway to leveraging Vectara’s robust search capabilities.
+
+- **Team Collaboration**: Enhance your project by inviting team members to the Vectara console. Assign roles, establish permissions, and cultivate a collaborative environment, all aimed at refining and perfecting your search solutions.
```

Comment:
Suggest to remove the last part of this sentence "all aimed at refinining and perfecting you search solutions." - not sure how it's tied to team collaboration.